src/main/webapp/app/shared/metis/message/message-reply-inline-input/message-reply-inline-input.component.html

```diff
@@ -22,23 +22,34 @@ <h4 class="alert-heading" jhiTranslate="artemisApp.messageWarning.headerText"></
                     <p class="mb-0" jhiTranslate="artemisApp.messageWarning.lastParagraph"></p>
                 </div>
             }
-            @if (editType === EditType.UPDATE) {
-                <button
-                    jhi-posting-button
-                    [buttonLabel]="'artemisApp.metis.cancel' | artemisTranslate"
-                    class="btn btn-sm btn-outline-secondary"
-                    (click)="isModalOpen.emit()"
-                ></button>
-                <button
-                    jhi-posting-button
-                    [buttonLoading]="isLoading"
-                    [disabled]="isLoading || !formGroup.valid"
-                    [buttonLabel]="'artemisApp.conversationsLayout.saveMessage' | artemisTranslate"
-                    class="btn btn-sm btn-outline-primary"
-                    id="save"
-                    type="submit"
-                ></button>
-            }
+            <div class="d-flex justify-content-between align-items-center">
+                @if (editType !== EditType.UPDATE) {
+                    <div class="direct-message-checkbox">
+                        <input type="checkbox" id="directMessageCheckbox" [checked]="sendAsDirectMessage()" (change)="toggleSendAsDirectMessage()" />
+                        <label for="directMessageCheckbox" jhiTranslate="artemisApp.conversationsLayout.sendAsDirectMessage"></label>
+                    </div>
+                }
+
+                <div class="d-flex gap-2">
+                    @if (editType === EditType.UPDATE) {
```

Comment:
I think here it would possibly make sense to only use the first condition and then use `@else`